packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -91,71 +132,86 @@ const BlockToolbar = ( { hideDragHandle } ) => {
 	const isSynced =
 		isReusableBlock( blockType ) || isTemplatePart( blockType );
 
-	const classes = classnames( 'block-editor-block-toolbar', {
+	// Shifts the toolbar to make room for the parent block selector.
+	const classes = classnames( 'block-editor-block-contextual-toolbar', {
+		'has-parent': hasParents && showParentSelector,
```

Comment:
Can we consolidate that into one selector?

Reply:
✅ Good call.